mlflow/ml-package-versions.yml

```diff
@@ -35,7 +35,7 @@ pytorch:
     requirements:
       ">= 0.0.0": ["torchvision", "scikit-learn"]
       ">= 1.8": ["transformers"]
-      "< 2.3": ["numpy<2", "transformers<=4.49.0"]
+      "< 2.3": ["numpy<2", "transformers<=4.49.0", "safetensors<0.6.0"]
```

Comment:
Only `models` tests failed. No need to add `safetensors<0.6.0` to `autologging`.